### DISCUSSION TITLE
Helm Chart for ssikit

### DISCUSSION THREAD

**Discussion by wistefan** [walt-id/.github#33] [Ideas / Feature Requests]
Hi,
is there already a helm-chart(or maybe kustomize) for waltid-ssikit? If not, I did build one and published it to one of our helm-repositories: https://github.com/i4Trust/helm-charts/tree/main/charts/vcwaltid
Since we are not the maintainers of the component, I would like to contribute it if you can tell me where:)
We are currently prepare to use Waltid as a part of the building blocks for our i4Trust framework, and provide tutorials based on helm-charts. You can see a WIP version here: https://github.com/i4Trust/tutorials/tree/add-vc/PacketDelivery-ReferenceExample/Data-Service-Provider#vcwaltid
Its mostly used for issuing credentials used in OIDC4VP/SIOP-2 flows for authn/z. 
Not directly related to Helm, but can you tell me about the release cycle of ssikit? When is the next stable release planned or do you provide non-volatile SNAPSHOT-tags? I was only able to find latest from master.

Best,
Stefan

**Comment by philpotisk** [comment]
Hi @wistefan the helm-chart is awesome. We were thinking of implementing this anyway, so we appreciate your contributions in this regard a lot. @severinstampler will probably merge it to our deployment scripts.

We are also aware of the issues with the current release cycle. We will update the release-process by the end of this month and will also provide a stable release. in the meantime it is probably better to fork the repo and use your own version of the SSI Kit, not our SNAPSHOT release as this one is changing to frequently.

**Reply by wistefan** [reply]
Hi,
nice to hear that you like it:) Thanks for the heads-up on the release process, I already have a retagged image on my repo. I will use it for the time beeing and then move back to the original.
Best

**Reply by waltkb** [reply]
Hi @wistefan, just wanted to inform you that we will be switching the SSI Kit (and all repositories in that regard) to a new versioning scheme in the very near future, thus you will be able to tag a specifc version of the SSI Kit and do no longer have to tag to a forked repository or similar.
